CRM/Member/BAO/Membership.php

```diff
@@ -1575,11 +1575,16 @@ public static function buildMembershipTypeValues($form, $membershipTypeID = [],
    * @return null|string
    */
   public static function getContactMembershipCount($contactID, $activeOnly = FALSE) {
-    CRM_Financial_BAO_FinancialType::getAvailableMembershipTypes($membershipTypes);
+    // @todo Move this check to financialtypeacls extension
+    $membershipTypes = \Civi\Api4\MembershipType::get(TRUE)
```

Comment:
@mattwire I think this whole function could be replaced by an api call - I suppose the gap is that we don't have v4 api membership.create  yet